### PR TITLE
MB-15227: fix playwright prime simulator

### PR DESCRIPTION
**Pull request by ahobson**
## Summary

Fix the playwright prime simulator tests in CI. They pass locally

### CODE REVIEW

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

File: .circleci/config.yml [hunk 1]
```diff
@@ -681,6 +681,8 @@ commands:
             export HTTP_ORDERS_SERVER_NAME=orderslocal
             export HTTP_PRIME_SERVER_NAME=primelocal
             export ENVIRONMENT=test
+            # disable for speed?
```
Comment (kylehilltruss) [inline .circleci/config.yml:684]: We should probably clarify this comment, or at least remove question tense ambiguity, for the sake of someone trying to understand it a year or so from now.

**Review by kylehilltruss — APPROVED** [review]
Change functionally looks good, aside from the comment below

File: .circleci/config.yml (hunk 1 above)
Comment (ahobson) [reply .circleci/config.yml]: Thank you, PR updated

**Pull request merged**